app/microwave-map.tsx

```diff
@@ -0,0 +1,80 @@
+import * as Haptics from 'expo-haptics';
+import * as Linking from 'expo-linking';
+import { Stack } from 'expo-router';
+import { View, Text, Platform } from 'react-native';
+import MapView, { Marker, Callout, CalloutSubview } from 'react-native-maps';
+
+import { Container } from '~/components/Container';
+import TopBar from '~/components/TopBar';
+import { MICROWAVE_LOCATION } from '~/data/MicrowaveLocations';
+import { COLORS } from '~/utils/colors';
+
+const MapMarkers = () => {
+  return (
+    <>
+      {MICROWAVE_LOCATION.map((location, index) => (
```

Comment:
nit-picky but rename this to `MICROWAVE_LOCATIONS`
```suggestion
      {MICROWAVE_LOCATION.map((location, index) => (
```